metadata_annotator/earthdata_varinfo_config.json

```diff
@@ -251,7 +263,7 @@
       "Applicability": {
         "Mission": "SMAP",
         "ShortNamePath": "SPL3SMP$",
-        "VariablePattern": "/Soil_Moisture_Retrieval_Data_(AM|PM)/(?!lc_type$).+$"
+        "VariablePattern": "/Soil_Moisture_Retrieval_Data_(AM|PM)/.*"
```

Comment:
~and no lc_types? (are these gone now from exclusions?)~ looks like it's hoisted?